### DISCUSSION TITLE
PCL2用的哪种Visual Studio模板啊

### DISCUSSION THREAD

**Discussion by SALTWOOD** [Meloong-Git/PCL#2393] [提问] [answered]
最近在做一些废柴，借鉴了一下PCL2的代码
我知道它是VB，想问问用的哪种模板
![1690816095545](https://github.com/Hex-Dragon/PCL2/assets/105980161/6640f6bf-01d3-4e85-acc5-44f1626bb7ba)

**Comment by Chiloven945** [comment]
PCL基于WPF

**Comment by LTCatt** [comment, marked as answer]
WPF 应用程序
